python/pyarrow/tests/test_fs.py

```diff
@@ -298,6 +300,99 @@ def subtree_s3fs(request, s3fs):
     )
 
 
+__minio_limited_policy = """{
```

Comment:
Single leading underscore should be sufficient for all of the "protected" variables and functions.

Reply:
Thanks!  I changed these to single underscore.